cmd/orb-server/startcmd/params.go

```diff
@@ -583,6 +590,21 @@ func getOrbParameters(cmd *cobra.Command) (*orbParameters, error) {
 		createDocumentStoreEnabled = enable
 	}
 
+	enableAllowedOriginsOptimizationStr, err := cmdutils.GetUserSetVarFromString(cmd, enableAllowedOriginsOptimizationFlagName, enableCreateDocumentStoreEnvKey, true)
```

Comment:
enableCreateDocumentStoreEnvKey typoe

Reply:
it should be enableAllowedOriginsOptimizationEnvKey

Reply:
Fixed